.travis.yml

```diff
@@ -9,10 +9,12 @@ addons:
     packages:
       - gcc-arm-none-eabi
 script:
-  - rustup target add thumbv7em-none-eabihf
+  - rustup target add thumbv7em-none-eabi thumbv7em-none-eabihf
+  - cargo build --manifest-path nrf52810-hal/Cargo.toml --target thumbv7em-none-eabi
```

Comment:
Oh wow, does the 810 really not have hardfloat?

Reply:
Nope haha